examples/deployment/deployment-cloudzone.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tcp-echo-cloud
+  namespace: cloudzone
+  labels:
+    app: tcp-echo-cloud
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: tcp-echo-cloud
+  template:
+    metadata:
+      labels:
+        app: tcp-echo-cloud
+    spec:
+      nodeName: k8s-node01
+      containers:
+        - name: tcp-echo-container
+          image: cjimti/go-echo:latest
+          imagePullPolicy: Always
+          env:
+            - name: TCP_PORT
+              value: "2701"
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+            - name: SERVICE_ACCOUNT
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.serviceAccountName
+          ports:
+            - name: tcp-echo-port
+              containerPort: 2701
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: tcp-echo-cloud-svc
+  namespace: cloudzone
+  labels:
+    app: tcp-echo-cloud
+spec:
+  selector:
+    app: tcp-echo-cloud
+  ports:
+    - name: tcp-0
+      protocol: TCP
+      port: 2701
+      targetPort: 2701
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: busybox-sleep-cloud
+  namespace: cloudzone
+  labels:
+    app: tcp-echo-cloud
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: tcp-echo-cloud
+  template:
+    metadata:
+      labels:
+        app: tcp-echo-cloud
+    spec:
+      nodeName: k8s-node01
```

Comment:
Same as above.

Reply:
OK, I'll revise it right away

Reply:
If the node name is not specified, the pod will be dispatched to the edge node。
![image](https://user-images.githubusercontent.com/88825053/134442435-cc97075c-8e32-4288-b1d3-7abee28caacd.png)

Reply:
Then we should try to use node affinity.

Reply:
OK, I'll modify it locally and test it

Reply:
So, should I label the cloud nodes myself,

The following is the label information of my node node

![image](https://user-images.githubusercontent.com/88825053/134443018-bc0c96e8-881e-4c7d-bb1c-921556bc3b36.png)

Reply:
You can refer to https://github.com/kubeedge/edgemesh/blob/main/build/agent/kubernetes/edgemesh-agent/07-daemonset-cloud.yaml

```
...
   affinity:
        nodeAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            nodeSelectorTerms:
              - matchExpressions:
                  - key: node-role.kubernetes.io/edge
                    operator: DoesNotExist
                  - key: node-role.kubernetes.io/agent
                    operator: DoesNotExist
...
```

Reply:
OK, I'll modify it and test it